### PR TITLE
ci: support @typescript-eslint/lines-between-class-members

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring
-   [ ] Code style update
-   [x] Build or CI related changes
-   [ ] Documentation content changes

## New behavior

Support quick fix

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -4,8 +4,13 @@ module.exports = {
     extends: ['@tinkoff/eslint-config-angular'],
     rules: {
         // TODO(splincode): enable later
```
Comment (waterplea) [inline .eslintrc.js:6]: Remove this line too please

File: projects/addon-charts/components/bar-set/test/bar-set.component.spec.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ describe('BarSet', () => {
     class TestComponent {
         @ViewChild(TuiBarSetComponent)
         readonly component: TuiBarSetComponent;
+
         readonly value = [10, 20, 30, 40];
         collapsed = false;
```
Comment (waterplea) [inline projects/addon-charts/components/bar-set/test/bar-set.component.spec.ts:20]: Why it didn't add one here?

**Review by waterplea — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (splincode) [reply .eslintrc.js]: done

**Comment by splincode** [comment]
@nsbarsukov @vladimirpotekhin please, review me)

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**